tests/integration/iop-remote-integration-test-defaults.yaml

```diff
@@ -14,3 +14,6 @@ spec:
     egressGateways:
     - name: istio-egressgateway
       enabled: false
+  values:
+    global:
+      externalIstiod: true
```

Comment:
should this be a part of the 'remote' profile?

Reply:
Yeah I considered that.

I would like to actually update the `remote` profile based on this: https://github.com/istio/istio.io/pull/15459

and based on how we properly separate (or not) `istio-discovery` and `istio-remote`.
 
Our current docs are not clear on which flags are required for which topology and which charts, and in fact `externalIstiod` is documented to do two contradictory things in a few spots, for the same chart (one of those documented uses is flatly wrong as per the current templates, but in general it's a bit messy.)

Reply:
See also https://github.com/istio/istio/pull/53016#discussion_r1775510055